geode-redis/src/main/java/org/apache/geode/redis/internal/executor/server/SlowlogExecutor.java

```diff
@@ -30,7 +27,21 @@
   @Override
   public RedisResponse executeCommand(Command command,
       ExecutionHandlerContext context) {
-    List<ByteArrayWrapper> values = new ArrayList<ByteArrayWrapper>();
-    return RedisResponse.array(values);
+    String subCommand = command.getStringKey();
+    if (subCommand.toLowerCase().equals("get")) {
+      return RedisResponse.emptyArray();
+    } else if (subCommand.toLowerCase().equals("len")) {
+      return RedisResponse.integer(0);
+    } else if (subCommand.toLowerCase().equals("reset")) {
+      return RedisResponse.ok();
+    }
+    throw new RedisParametersMismatchException(wrongNumberOfArgumentsErrorMessage());
+  }
+
+  public String wrongNumberOfArgumentsErrorMessage() {
+    String result;
+    result = String
+        .format("Unknown subcommand or wrong number of arguments for 'len'. Try SLOWLOG HELP.");
+    return result;
```

Comment:
It seems kind of weird that we have this hard coded to `len`... right now if I type in `slowlog sjls`, I get `ERR Unknown subcommand or wrong number of arguments for 'len'. Try SLOWLOG HELP.` instead of `ERR Unknown subcommand or wrong number of arguments for 'jsls'. Try SLOWLOG HELP.` It also might be good to add this Error to the `RedisConstants` class.

Reply:
Also fixed when we refactored. Thanks!